remote-access/ssh/android.md

```diff
@@ -0,0 +1,63 @@
+# SSH USING ANDROID
+
+To use SSH on your mobile device you need to download a client. There are several different good quality clients available, such as [Serverauditor](http://www.serverauditor.com), [JuiceSSH](https://juicessh.com/), and [Connectbot](https://connectbot.org/). For the sake of this tutorials we'll use Serverauditor, because it's a popular cross platform SSH client. But the principle is similar and applicable to other clients as well. 
+
+
+##1. Add your Raspberry pi as a host
+
+Download Serverauditor [Google Play](https://play.google.com/store/apps/details?id=com.server.auditor.ssh.client), if you haven’t it installed yet. And click to open the app.
+
+The app should open showing ‘no hosts’. To get started, you should tap the blue + button in the lower left corner. Then tap ‘New Host’. 
+
+![Serverauditor ‘New Host’ configuration](images/ssh-android-config.png)
+
+Enter an `alias`, Raspberry Pi, for example. Then enter the IP address under `hostname`. Fill out the `username` and `password` and hit the tick (✓) in the top right corner. 
+
+If you don't know the IP address just type `hostname -I` in the Raspberry Pi. See more [methods](https://www.raspberrypi.org/documentation/remote-access/ip-address.md) of finding your IP address. The default login for Raspbian is `pi` with the password `raspberry`.
```

Comment:
As earlier (and also elsewhere in this PR) these should all be relative links rather than absolute.